### PR TITLE
Split config for 3.10 from master in autoscaler repo

### PR DESCRIPTION
**Pull request by paulfantom**
SSIA

/assign @ingvagabund

### CODE REVIEW

File: ci-operator/jobs/openshift/kubernetes-autoscaler/openshift-kubernetes-autoscaler-postsubmits.yaml [hunk 1]
```diff
@@ -30,33 +30,3 @@ postsubmits:
           requests:
             cpu: 10m
       serviceAccountName: ci-operator
-  - agent: kubernetes
```
Comment (ingvagabund) [inline ci-operator/jobs/openshift/kubernetes-autoscaler/openshift-kubernetes-autoscaler-postsubmits.yaml]: Please open a separate PR if you want to move the configuration from `openshift-kubernetes-autoscaler-postsubmits.yaml` to `openshift-kubernetes-autoscaler-postsubmits-release-3.10.yaml`.

File: ci-operator/jobs/openshift/kubernetes-autoscaler/openshift-kubernetes-autoscaler-presubmits.yaml [hunk 2]
```diff
@@ -32,35 +32,12 @@ presubmits:
...
-          limits:
-            cpu: 500m
-          requests:
-            cpu: 10m
-      serviceAccountName: ci-operator
-    trigger: ((?m)^/test( all| unit),?(\s+|$))
+    context: ci/openshift-jenkins
+    labels:
+      master: ci.dev.openshift.redhat.com
+    name: test_branch_autoscaler
```
Comment (ingvagabund) [inline ci-operator/jobs/openshift/kubernetes-autoscaler/openshift-kubernetes-autoscaler-presubmits.yaml]: This gets run over `openshift/online-registration` repo, not `openshift/kubernetes-autoscaler`

**Comment by paulfantom** [comment]
@ingvagabund repurposed this PR.

**Pull request merged**